API/NoDocs/Microsoft.Extensions.DependencyInjection/FakeRedactionServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,12 @@
+// Assembly 'Microsoft.Extensions.Compliance.Testing'
+
+using System;
+using Microsoft.Extensions.Compliance.Testing;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+public static class FakeRedactionServiceCollectionExtensions
+{
+    public static IServiceCollection AddFakeRedaction(this IServiceCollection services);
+    public static IServiceCollection AddFakeRedaction(this IServiceCollection services, Action<FakeRedactorOptions> configure);
+}
```

Comment:
Is it missing `public static IServiceCollection AddFakeRedaction(this IServiceCollection services, IConfigurationSection section);`